opencti-platform/opencti-graphql/src/domain/backgroundTask-common.js

```diff
@@ -244,8 +246,21 @@ export const checkActionValidity = async (context, user, input, scope, taskType)
   }
 };
 
-export const createDefaultTask = (user, input, taskType, taskExpectedNumber, scope = undefined) => {
+const createWorkForBackgroundTask = async (context, taskId, connectorId) => {
+  const connector = { internal_id: connectorId, connector_type: ConnectorType.ExternalImport };
+  const args = { background_task_id: taskId, receivedTime: now() };
+  return createWork(context, SYSTEM_USER, connector, `background task @ ${now()}`, connector.internal_id, args);
```

Comment:
I don't see how the user is transfered to execute the work with the accurates permissions, should't be send in args ?

Reply:
Its not in the work but directly inside the sendResultToQueue specifiying the applicant_id